src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/sql.y

```diff
@@ -748,10 +759,22 @@ func forceEOF(yylex interface{}) {
 
 %type	<setExprs>
 	set_list
```

Comment:
Is the `set_list` here support syntax `SET variable = expr [, variable = expr] ...` ? 
Or we currently just support `set variable=expr` ? If so, the commit comment should be changed.

Reply:
yes,  support